### PR TITLE
Fixed #32

### LINKED ISSUE #32: Back navigation when viewing a file within a folder returns to the overall screen rather than the folder
#### General information

* **Device:**  Oneplus 3T
* **Android Version:**  7.1.2
* **App version:**  v0.1.3

#### Steps to reproduce

1. Ensure the option 'Remember last folder location on startup' is **unchecked**
2. View file (either view or edit mode) within a folder
3. Press back button

Additional setting change from default: 'Preview first when opening notes' is **checked**

#### Expected result
**What is the expected output?** 
Return to the file listing within the folder the file was in

**What do you see instead?**
The contents of storage folder selected in options

**Additional Info/Comments**  
It appears that the option to 'Remember last folder on startup' being unchecked additionally causes Markor to not remember the last folder when exiting the view/edit mode.

### PR DESCRIPTION
**Pull request by beast-incarnate**
Fixed by keeping a track of Last Opened Directory even if the user has deselected the option and checking for the same at the start of the MainActivity.

### CODE REVIEW

File: app/src/main/java/net/gsantner/markor/activity/MainActivity.java [hunk 1]
```diff
@@ -71,13 +71,18 @@
 
     private boolean _doubleBackToExitPressedOnce;
 
+
```
Comment (gsantner) [inline app/src/main/java/net/gsantner/markor/activity/MainActivity.java]: can be removed

File: app/src/main/java/net/gsantner/markor/activity/NoteActivity.java [hunk 2]
```diff
@@ -74,6 +74,7 @@
     private String _initialContent = "";
     private String _initialFileName = "";
 
+
```
Comment (gsantner) [inline app/src/main/java/net/gsantner/markor/activity/NoteActivity.java]: no neee

File: app/src/main/java/net/gsantner/markor/activity/PreviewActivity.java [hunk 3]
```diff
@@ -57,6 +57,7 @@
     private boolean _isEditIncoming = false;
     private MarkDownRenderer _renderer = new MarkDownRenderer();
 
+
```
Comment (gsantner) [inline app/src/main/java/net/gsantner/markor/activity/PreviewActivity.java]: no need

File: app/src/main/java/net/gsantner/markor/activity/MainActivity.java (hunk 1 above)
Comment (beast-incarnate) [reply app/src/main/java/net/gsantner/markor/activity/MainActivity.java]: Should I make another PR with the necessary changes?

Comment (gsantner) [reply app/src/main/java/net/gsantner/markor/activity/MainActivity.java]: no, just reset those lines and make commit :)

**Comment by gsantner** [comment]
thanks! merging on PC then

**Comment by gsantner** [comment]
Thanks! here are some more if you are interested: <https://github.com/search?&q=label%3Ahacktoberfest+user%3Agsantner>

**Pull request merged**